updated the ISBN description for additonal clarity

blakeashleyjr opened the pull request:
This PR is based on the discussion of [this issue.](https://github.com/exercism/problem-specifications/issues/1975)

When using the default code blocks with the text language, the subscript characters are removed. I switched them to block quotes to preserve the subscripts and everything looks good because it is only text anyways. Let me know if this violates a standard somewhere

blakeashleyjr commented:
The linter failed because it is confusing the "*"s with emphasis marks when they represent multiplication symbols.

SaschaMann commented on exercises/isbn-verifier/description.md:
```diff
@@ -5,21 +5,21 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
+The ISBN-10 format is 9 digits (0 to 9, represented below with a subscript) plus one check character (either a digit or an X only). In the case the check character is an X, this represents the value '10'. These may be communicated with or without hyphens, and can be checked for their validity by the following formula:
+
+
+> (1₁ * 10 + 2₂ * 9 + 3₃ * 8 + 4₄ * 7 + 5₅ * 6 + 6₆ * 5 + 7₇ * 4 + 8₈ * 3 + 9₉ * 2 + 10₁₀ * 1) mod 11 == 0
```
This notation is really confusing. I only know digits with indices like this where the index clarifies which base the number is in. I think it was suggested that these should be `d` with subscripts?

SaschaMann commented:
> The linter failed because it is confusing the "*"s with emphasis marks when they represent multiplication symbols.

You can escape them with a backslash. But I don't understand why the text block removes subscripts. Did you use the Unicode subscripts or `<sub>`?

blakeashleyjr commented:
Thanks @SaschaMann, I corrected the markdown according to your suggestion and switched the digits to a "d" for more clarity. Let me know what you think

The text block removed the Unicode subscripts and <sub> tags for me in VSCode. Interestingly, the tags do work outside of a text block though:

![image](https://user-images.githubusercontent.com/56891775/156862599-c0f3113f-31f1-4b50-ab04-0ad5aff72ba7.png)

I can switch to using <sub> tags but they are likely less supported than the Unicode characters.

angelikatyborska commented:
> The text block removed the Unicode subscripts [...] for me in VSCode

Are you sure? They do work for me (a fresh VSC installation on a m1 mac). Here's screenshots of VSC and RubyMine rendering the subscript unicode characters in a text code block:
<img width="1207" alt="Screenshot 2022-03-14 at 17 16 32" src="https://user-images.githubusercontent.com/7852553/158215424-4f0d842a-eee0-4422-a0c2-1e58ba99e12b.png">
<img width="1100" alt="Screenshot 2022-03-14 at 17 16 53" src="https://user-images.githubusercontent.com/7852553/158215446-f65803dc-ed01-4cf4-baeb-edcc6198948c.png">

SaschaMann commented on exercises/isbn-verifier/description.md:
````diff
@@ -7,19 +7,15 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
-```text
-(x1 * 10 + x2 * 9 + x3 * 8 + x4 * 7 + x5 * 6 + x6 * 5 + x7 * 4 + x8 * 3 + x9 * 2 + x10 * 1) mod 11 == 0
-```
+> (d₁ \* 10 + d₂ \* 9 + d₃ \* 8 + d₄ \* 7 + d₅ \* 6 + d₆ \* 5 + d₇ \* 4 + d₈ \* 3 + d₉ \* 2 + d₁₀ \* 1) mod 11 == 0
````
```suggestion
```text
(d₁ * 10 + d₂ * 9 + d₃ * 8 + d₄ * 7 + d₅ * 6 + d₆ * 5 + d₇ * 4 + d₈ * 3 + d₉ * 2 + d₁₀ * 1) mod 11 == 0
```
```

SaschaMann commented on exercises/isbn-verifier/description.md:
````diff
@@ -7,19 +7,15 @@ numbers. These normally contain dashes and look like: `3-598-21508-8`
-```text
-(3 * 10 + 5 * 9 + 9 * 8 + 8 * 7 + 2 * 6 + 1 * 5 + 5 * 4 + 0 * 3 + 8 * 2 + 8 * 1) mod 11 == 0
-```
+> (3 \* 10 + 5 \* 9 + 9 \* 8 + 8 \* 7 + 2 \* 6 + 1 \* 5 + 5 \* 4 + 0 \* 3 + 8 \* 2 + 8 \* 1) mod 11 == 0
````
```suggestion
```text
(3 * 10 + 5 * 9 + 9 * 8 + 8 * 7 + 2 * 6 + 1 * 5 + 5 * 4 + 0 * 3 + 8 * 2 + 8 * 1) mod 11 == 0
```
```

SaschaMann reviewed (commented):
> Are you sure? They do work for me (a fresh VSC installation on a m1 mac). Here's screenshots of VSC and RubyMine rendering the subscript unicode characters in a text code block:

Yeah they work fine for me too.

When we get the equation support that was promised as a launch feature, this would be a great candidate to convert to that.

blakeashleyjr commented:
Thank you @SaschaMann and @angelikatyborska for testing that. I do have a couple markdown specific plugins enabled that may be interfering with the text box specifically.

I changed the description according to your suggestions. Let me know what you think now.

SaschaMann reviewed (approved):
Thanks!

@kotp talking about the website/v3 launch.
